Engine/Component/ComponentScript.cpp

```diff
@@ -0,0 +1,71 @@
+#include "ComponentScript.h"
+#include "Main/Application.h"
+#include "Module/ModuleRender.h"
+#include "Main/GameObject.h"
+#include "Module/ModuleScriptManager.h"
+#include "Script/Script.h"
+#include "imgui.h"
```

Comment:
It is a very fancy stylish thing but includes should be in alphabetic order from the beggining so it should be:

#include "ComponentScript.h"
#include "Main/Application.h"
#include "Main/GameObject.h"
#include "Module/ModuleRender.h"
#include "Module/ModuleScriptManager.h"
#include "Script/Script.h"
#include "imgui.h"

Reply:
Done